src/components/dashboard/sidebar/DashboardSidebar.jsx

```diff
@@ -32,25 +32,32 @@ class DashboardSidebar extends React.Component {
   }
 
   renderOfferSummaryCard() {
-    const { offersCount } = this.props;
-    return (
-      <SidebarCard
-        title="Assigned courses left to redeem"
-        buttonText="Find a course in the catalog"
-        textClassNames={offersCount ? 'big-number' : ''}
-        buttonLink="/search"
-        linkIsLocal
-      >
-        {offersCount || 'You currently have no coupons to redeem.'}
-      </SidebarCard>
-    );
+    const { enterpriseConfig: { name } } = this.context;
```

Comment:
interesting. I've never actually seen a sub-object definition from a prop/context before.

Reply:
I hadn't seen it until recently! And I was super into it and am now using it everywhere.